src/main/java/com/direwolf20/buildinggadgets/common/util/blocks/RegionSnapshot.java

```diff
@@ -151,7 +151,7 @@ public static RegionSnapshot deserialize(NBTTagCompound tag) {
             int stateID = frame & 0xffffff;
             // The serialized number will be ONE FEWER then the actual amount
             // - see the serialization for +1
-            int streaks = (frame >> 24) + 1;
+            int streaks = (frame >>> 24) + 1;
```

Comment:
What happened before is `>>` doesn't include the sign bit, which we used for storing data.

Reply:
Oh these typos... Killing us...